docs/en/setup/backend/backend-apisix-monitoring.md

```diff
@@ -0,0 +1,45 @@
+# APISIX monitoring
+## APISIX  performance from `apisix prometheus plugin`
+SkyWalking leverages OpenTelemetry Collector to transfer the metrics to
+[OpenTelemetry receiver](opentelemetry-receiver.md) and into the [Meter System](./../../concepts-and-designs/meter.md).
+
+### Data flow
+1. [APISIX Prometheus plugin](https://apisix.apache.org/docs/apisix/plugins/prometheus/) collects metrics data from APSIX.
+2. OpenTelemetry Collector fetches metrics from [APISIX Prometheus plugin](https://apisix.apache.org/docs/apisix/plugins/prometheus/) via Prometheus Receiver and pushes metrics to SkyWalking OAP Server via the OpenCensus gRPC Exporter or OpenTelemetry gRPC exporter.
+3. The SkyWalking OAP Server parses the expression with [MAL](../../concepts-and-designs/mal.md) to filter/calculate/aggregate and store the results.
+
+### Set up
+1. Enable APISIX [APISIX Prometheus plugin](https://apisix.apache.org/docs/apisix/plugins/prometheus/) .
+2. Set up [OpenTelemetry Collector ](https://opentelemetry.io/docs/collector/getting-started/#docker). For details on Prometheus Receiver in OpenTelemetry Collector, refer to [here](../../../../test/e2e-v2/cases/apisix/otel-collector/otel-collector-config.yaml).
+3. Config SkyWalking [OpenTelemetry receiver](opentelemetry-receiver.md).
+
+### MySQL Monitoring
+[APISIX prometheus plugin](https://apisix.apache.org/docs/apisix/plugins/prometheus/) provide multiple dimensions metrics for APISIX server , node , route , etc. 
+Accordingly, Skywalking observes the status, payload, and latency of the APISIX server, which is cataloged as a `LAYER: APISIX` `Service` in the OAP. Meanwhile, the server node would be recognized as `instance, and the route rule would be recognized as `endpoint`.
+
+#### Supported Metrics 
+| Monitoring Panel | Unit | Metric Name |Catalog | Description | Data Source |
+|-----|------|-----|-----|-----|-----|
+|HTTP status(per second)  |  | meter_apisix_sv_http_status |Service | The rate of http status trend| APISIX prometheus plugin|
```

Comment:
```suggestion
|HTTP status trend  |  | meter_apisix_sv_http_status |Service | The increment rate of the status of HTTP requests| APISIX Prometheus plugin|
```

Please make case sensitivity correct. Also, `per second` in the name is very confusing. The rate of trend doesn't make sense.

Reply:
All followings should be updated too.